.github/workflows/prod.yaml

```diff
@@ -4,42 +4,37 @@ on:
     - cron: "0 */24 * * *"
   workflow_dispatch:
 env:
-        SAUCE_USERNAME: ${{ secrets.SAUCE_USERNAME }}
-        SAUCE_ACCESS_KEY: ${{ secrets.SAUCE_ACCESS_KEY }}
-        SAUCE_JOB: "Alloy Prod Workflow"
-        SAUCE_CAPABILITIES_OVERRIDES_PATH: 'sauceLabsCapabilities.json'
-        ALLOY_ENV: prod
+  SAUCE_USERNAME: ${{ secrets.SAUCE_USERNAME }}
+  SAUCE_ACCESS_KEY: ${{ secrets.SAUCE_ACCESS_KEY }}
+  SAUCE_JOB: "Alloy Prod Workflow"
+  SAUCE_CAPABILITIES_OVERRIDES_PATH: 'sauceLabsCapabilities.json'
+  ALLOY_ENV: prod
 
 jobs:
   alloy-prod-e2e:
     name: "Cron: Prod E2E Tests"
     runs-on: ubuntu-latest
     steps:
-      - name: "Get latest Alloy Release"
-        id: last_release
-        uses: InsonusK/get-latest-release@v1.0.1
+      - uses: actions/checkout@v2
         with:
-          myToken: ${{ github.token }}
-          exclude_types: "draft|prerelease"
-      - uses: actions/checkout@v2.3.3
-        with:
-          ref: ${{ steps.last_release.outputs.tag_name }}
-      - uses: actions/cache@v2
-        id: npm-cache
-        with:
-          path: '**/node_modules'
-          key: ${{ runner.os }}-modules-${{ hashFiles('**/package-lock.json') }}
-      - name: Install dependencies
-        if: steps.npm-cache.outputs.cache-hit != 'true'
-        run: npm ci
-      - name: Build
-        run: npm run test:functional:build:prod
-      - name: Run TestCafe Tests
-        run: npm run test:functional:ci:prod
+          # Fetch all the branches and tags in the repo
+          fetch-depth: 0
+      # install these dependencies so getTestingTags will work
+      - run: npm install -g @octokit/rest semver
+      - run: |
+          for TAG in $(./scripts/getTestingTags.js)
+          do
+            git checkout tags/$TAG
+            npm ci
+            npm run test:functional:build:prod
+            export ALLOY_PROD_VERSION=`node -e 'console.log(require("./package.json").version)'`
+            npx testcafe -q -c 5 'saucelabs:Chrome@latest:macOS 11.00','saucelabs:IE@11.285:Windows 10','saucelabs:Firefox@latest:Windows 10','saucelabs:Safari@latest:macOS 11.00'
```

Comment:
I noticed we're doing this instead of calling `npm run test:functional:ci:prod`. The `test:functional:ci:prod` npm script sets `SAUCE_CAPABILITIES_OVERRIDES_PATH`. Should we be setting SAUCE_CAPABILITIES_OVERRIDES_PATH here?